test/TEST-02-CRYPTSETUP/test.sh

```diff
@@ -86,10 +79,10 @@ EOF
 }
 
 test_cleanup() {
-    umount $TESTDIR/root/var 2>/dev/null
-    [[ -b /dev/mapper/varcrypt ]] && cryptsetup luksClose /dev/mapper/varcrypt
-    umount $TESTDIR/root 2>/dev/null
-    [[ $LOOPDEV ]] && losetup -d $LOOPDEV
+    umount $TESTDIR/root/var 2>/dev/null || true
+    [ ! -b /dev/mapper/varcrypt ] || cryptsetup luksClose /dev/mapper/varcrypt
+    umount $TESTDIR/root 2>/dev/null || true
+    [ -z "$LOOPDEV" ] || losetup -d $LOOPDEV || true
```

Comment:
That's a matter of stylistic preference, of course. Most people I know couldn't tell me the intricacies of the bash specific `[[`, and we don't actually need them for most of the checks here. Also, unquoted arguments like `[[ $LOOPDEV ]]` make me cringe, as they are pretty much always a trap. I can revert these parts if reviewers prefer it that way, it's not important enough to waste time arguing about.